Sources/RabbitMq/Connection/RetryingConnection.swift

```diff
@@ -23,8 +23,10 @@ public actor RetryingConnection: Connection, Service {
     private(set) var connectionAttempts: Int = 0
 
     // Protocol conformances
-    public let logger: Logger  // shared to users of Connection
     public let connectionPollingInterval: Duration
+    public var logger: Logger {
+        get async { await basicConnection.logger }
```

Comment:
Doing an async call every time `logger` is used may not be ideal, but if it doesn't happen too often, I guess it's fine.